README.md

```diff
@@ -24,25 +29,27 @@ sudo apt-get install -y zsh git vim
 sudo chsh # change shell to /usr/bin/zsh and restart
 sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
 git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
+git clone https://github.com/zsh-users/zsh-autosuggestions ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autosuggestions
+
+cat > $HOME/.zshrc << END
+plugins=(git microk8s zsh-syntax-highlighting zsh-autosuggestions)
```

Comment:
Don't know this zsh-autosuggestions plugin, and I don't think it's mandatory on the on prem production machines

Reply:
I thought that you added the plugins to make the dev's workflow easier, to run this in prod mode I don't think that we need these plugins at all...